route_guide/route_guide_server.cc

```diff
@@ -55,133 +64,235 @@ float GetDistance(const Point& start, const Point& end) {
   float lon_2 = end.longitude() / kCoordFactor;
   float lat_rad_1 = ConvertToRadians(lat_1);
   float lat_rad_2 = ConvertToRadians(lat_2);
-  float delta_lat_rad = ConvertToRadians(lat_2-lat_1);
-  float delta_lon_rad = ConvertToRadians(lon_2-lon_1);
+  float delta_lat_rad = ConvertToRadians(lat_2 - lat_1);
+  float delta_lon_rad = ConvertToRadians(lon_2 - lon_1);
 
-  float a = pow(sin(delta_lat_rad/2), 2) + cos(lat_rad_1) * cos(lat_rad_2) *
-            pow(sin(delta_lon_rad/2), 2);
-  float c = 2 * atan2(sqrt(a), sqrt(1-a));
+  float a = pow(sin(delta_lat_rad / 2), 2)
+      + cos(lat_rad_1) * cos(lat_rad_2) * pow(sin(delta_lon_rad / 2), 2);
+  float c = 2 * atan2(sqrt(a), sqrt(1 - a));
   int R = 6371000; // metres
 
   return R * c;
 }
 
-std::string GetFeatureName(const Point& point,
-                           const std::vector<Feature>& feature_list) {
+// Just return the name of feature in feature_list vector
+std::string GetFeatureName(
+    const Point& point,
+    const std::vector<Feature>& feature_list) {
   for (const Feature& f : feature_list) {
-    if (f.location().latitude() == point.latitude() &&
-        f.location().longitude() == point.longitude()) {
+    if (
+        f.location().latitude() == point.latitude()
+        && f.location().longitude() == point.longitude()) {
       return f.name();
     }
```

Comment:
You can probably simplify this with [protobuf's `MessageDifferencer`](https://stackoverflow.com/questions/3228107/google-protocol-buffers-compare):
```suggestion
    if (MessageDifferencer.Equals(f.location(), point) {
      return f.name();
    }
```

Reply:
... But if somehow `MessageDifferencer` isn't easy to use in our code, don't make too much of an effort for it. It's only a minor improvement here.